backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -76,20 +76,29 @@ def get_collections_curation(visibility: str, token_info: dict):
     collections = Collection.list_collections_for_curation(g.db_session, visibility)
     allowed_collections = []
     for collection in collections:
-        owner = collection["owner"]
-        if is_user_owner_or_allowed(token_info, owner):
-            collection["access_type"] = "WRITE"
-        elif collection["visibility"] == CollectionVisibility.PRIVATE:
-            continue
-        else:
-            collection["access_type"] = "READ"
-        del collection["owner"]  # Don't actually want to return 'owner' in response
-        collection["collection_url"] = f"{CorporaConfig().collections_base_url}/collections/{collection['id']}"
-        allowed_collections.append(collection)
+        if reshape_for_curation_api_and_is_allowed(collection, token_info):
+            allowed_collections.append(collection)
+
+    import pprint
+
+    pprint.pprint(allowed_collections)
 
     return jsonify({"collections": allowed_collections})
 
 
+@dbconnect
+def get_collection_uuid_curation(collection_uuid: str, token_info: dict):
```

Comment:
remove curation from the name or change to `get`, and move this to _backend/corpora/lambdas/api/v1/curation/collections/collection_uuid/actions.py_